HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -51,6 +51,21 @@ public ActionResult getPublishedFeedbacks()
             }
         }
 
+        [HttpPut]
+        public ActionResult PublishFeedback(FeedbackDTO feedbackDTO)
```

Comment:
This isn't really the path I'd expect this to be at when it comes to [RESTful resource naming](https://restfulapi.net/resource-naming/), put at this path would imply updating the entire list. I would expect something like put at /feedback/{id}, or even put or post at /feedback/{id}/publish since it's not supposed to be a general update.